mail/postfix/pkg-descr

```diff
@@ -2,4 +2,13 @@ Postfix attempts to be fast, easy to administer, and secure.
 The outside has a definite Sendmail-ish flavor, but the inside
 is completely different.
 
+
+Plugin Changelog
+================
+
+1.7
+
+* Add Address Rewriting, allows to rewrite e.g. @example.com to @example.net.
```

Comment:
You don't need it but in case of nginx the changelog is too big for the release note in the forum and the other places, that's why it has its own (1/5 would be only the nginx plugin).
But it is your decision.

Reply:
Yesterday I asked @fichtner for a plugin announce (read-only) forum so we can announce new features in more detail, but we agreed to use your approach via changelog in pkg-descr as noone needs to go to the forums :)